### DISCUSSION TITLE
Don't sync Tabby data

### DISCUSSION THREAD

**Discussion by justc0de-it** [Eugeny/tabby#9166] [General]
When I try use my token Tabby send me error: AxiosError: Request failed with status code 401. When I try go to site https://app.tabby.sh/ I see this error:

Error: Forbidden
Your client does not have permission to get URL / from this server.

**Comment by Clem-Fern** [comment]
Hey @justc0de-it 

app.tabby.sh has been discontinued.
Detail in #9131

**Reply by justc0de-it** [reply]
How create own Tabby Web instance?

**Reply by Clem-Fern** [reply]
You can take a look here: https://github.com/Eugeny/tabby-web
It's the source of app.tabby.sh

**Reply by justc0de-it** [reply]
Ok, thx

**Comment by niceit** [comment]
I may reborn this one, by the way there are a lot of incoming works because the tabby vault is conflicting with this one

https://github.com/niceit/tabby-cloud-sync-settings

Let me know if you guys are interested in it.

**Reply by weikai** [reply]
I actually already switched back to the plugin even though it's deprecated.
